src/include/migraphx/op/dequantizelinear.hpp

```diff
@@ -37,6 +37,14 @@ namespace op {
 
 struct dequantizelinear
 {
+
+    value attributes() const {
+        // Note: point_op attribute is not used in this op. Instead, in
+        // gpu compilation pipeline, rewrite_quantization will be invoked
+        // from generate_pointwise() to rewrite this op.
+        return {{"pointwise", true}, {"point_op", ""}};
```

Comment:
Just remove the `point_op` attribute.

Reply:
If we do keep the `point_op` we should make it a `static_assert`, so any codegen can have a clearer message if it tries to generate this operator:

```cpp
return {{"pointwise", true}, {"point_op", R"(static_assert(false, "dequantizelinear does not have a point_op"))" }};
```

Of course, we already through an error if `point_op` is missing, which I think is cleaner then doing this `static_assert` injection. So I think it would be better to remove the `assert`.

Reply:
Sounds good, seems like a reasonable decision. Got rid of the assert as well as attribute in 6040c79751f850f33d9cb7c53394dfce38160d97